src/utils.ts

```diff
@@ -40,4 +45,81 @@ export function fixRemoteUrl(remoteAddress: string, remotePort: number, target:
         }
     }
     return target;
+}
+
+
+export const enum Platform {
+    Windows, OSX, Linux
+}
+
+export function getPlatform(): Platform {
+    const platform = os.platform();
+    return platform === 'darwin' ? Platform.OSX :
+        platform === 'win32' ? Platform.Windows :
+            Platform.Linux;
+}
+
+export function existsSync(path: string): boolean {
+    try {
+        fs.statSync(path);
+        return true;
+    } catch (e) {
+        return false;
+    }
+}
+
+export function launchLocalChrome(targetUrl: string) {
+    const chromePath = getPathToChrome();
+    const chromeArgs = ['--remote-debugging-port=9222'];
```

Comment:
Should the port be customizable via settings?

Reply:
Updated.